src/qililab/instruments/rohde_schwarz/sgs100a.py

```diff
@@ -168,6 +177,14 @@ def set_parameter(self, parameter: Parameter, value: ParameterValue, channel_id:
             if self.is_device_active():
                 self.device.IQ_state(self.iq_modulation)
             return
+        if parameter == Parameter.OPERATION_MODE:
+            if value not in ("normal", "bypass"):
+                raise ParameterNotFound(self, parameter)
+            self.settings.operation_mode = value
```

Comment:
Missing a str to fit the correct format
```suggestion
            self.settings.operation_mode = str(value)
```

Reply:
done

Reply:
I didn't see the last commit (I wonder why), your implementation with `str(value).lower()` is better, maybe you can omit a line if you replace value by this whenever is called but otherwise is ok the way you did it